### PR TITLE
Add model name to meta title

### PR DESCRIPTION
**Pull request by carkod**
## Done

Add model name as requested in the issue 
## QA

- Go to https://ubuntu-com-11921.demos.haus/certified/202202-29982 and check that metadata title contains the form factor as reported in the issue

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/11826

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/certified/model-details.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends "templates/base.html" %}
 
-{% block title %} {{ vendor }} {{ model }} certified with Ubuntu{% endblock %}
+{% block title %} {{ vendor }} {{ model }} {{ form_factor }} certified with Ubuntu{% endblock %}
```
Comment (sowasred2012) [inline templates/certified/model-details.html:3]: The [issue](https://github.com/canonical/ubuntu.com/issues/11826) asks that the page title be:
>Lenovo Think Centre Neo30a 22 certified with Ubuntu

but the demo shows:
>Lenovo  all-in-one certified with Ubuntu

Looking at the `h1` on the page, I think the tags should be:
```suggestion
{% block title %} {{ vendor }} {{ name }} certified with Ubuntu{% endblock %}
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]

File: templates/certified/model-details.html (hunk 1 above)
Comment (sowasred2012) [reply templates/certified/model-details.html]: @carkod maybe I was wrong, the page is 500ing now.

Comment (carkod) [reply templates/certified/model-details.html]: Right, thanks.

**Review by sowasred2012 — APPROVED** [review]
LGTM

**Pull request merged**